src/scores/probability/crps_impl.py

```diff
@@ -174,35 +174,39 @@ def crps_cdf(
     is used as an index for predictive CDF values. Various techniques are used to
     interpolate CDF values between indexed thresholds.
 
-    Given
-        - a predictive CDF `fcst` indexed at thresholds by variable x,
-        - an observation in CDF form `obs_cdf` (i.e., obs_cdf(x) = 0 if x < obs and 1 if x >= obs),
+    Given:
+        - a predictive CDF `fcst` indexed at thresholds by variable x
+        - an observation in CDF form `obs_cdf` (i.e., obs_cdf(x) = 0 if x < obs and 1 if x >= obs)
         - a `threshold_weight` array indexed by variable x,
 
     The threshold-weighted CRPS is given by:
-        `CRPS = integral(threshold_weight(x) * (fcst(x) - obs_cdf(x))**2)`, over all thresholds x.
-    The usual CRPS is the threshold-weighted CRPS with `threshold_weight(x) = 1` for all x.
-    This can be decomposed into an over-forecast penalty
+        - `CRPS = integral(threshold_weight(x) * (fcst(x) - obs_cdf(x))**2)`, over all thresholds x.
```

Comment:
Update all of these equations in this docstring to render as equations like you did for CRPS for ensembles

Reply:
I have attempted this but I'm not familiar enough with the correct formula to know for sure if I have represented every part of it properly in the mathjax. If you can review, that would be greatly appreciated.